workflows/funcscan.nf

```diff
@@ -99,89 +97,34 @@ workflow FUNCSCAN {
     // Some tools require annotated FASTAs
     // TODO only execute when we run tools that require prokka as input, e.g.
     // if ( params.run_bgc_tool1 | params.run_bgc_tool2 | params.run_bgc_tool3 ) etc.
-    if ( params.run_arg_deeparg ) {
+    if ( params.run_arg_screening & !params.arg_skip_deeparg ) {
```

Comment:
Single `&` is the bitwise AND operator. It works in this case but better to use the logical operator `&&`.
https://groovy-lang.org/operators.html#_logical_operators

```suggestion
    if ( params.run_arg_screening && !params.arg_skip_deeparg ) {
```

Reply:
oops, yup -  normally use it. Typo here! Good catch thanks!

Reply:
Didn't know about that it was bitewise, thanks for the link!